client/my-sites/plan-features/index.jsx

```diff
@@ -679,6 +679,14 @@ export class PlanFeatures extends Component {
 		const description = feature.getDescription
 			? feature.getDescription( abtest, this.props.domainName )
 			: null;
+
+		const recordEvent = feature.getTrackingLabel
+			? () =>
+					this.props.recordTracksEvent( 'calypso_plan_link_click', {
```

Comment:
My $0.02: I find this event name confusing - it's not a plan link, it's a plan feature link, no? Should this be `calypso_plan_feature_link_click` instead?

Reply:
As I understand it, it's intentional. Here's @lucasartoni's [explanation](https://github.com/Automattic/wp-calypso/pull/37442/files#r344214681):

> The eventual use of that event is to identify where people leave the page. Let's say there is one of those links that attracts a lot of clicks, it would be interesting to find out why.

I can see that we're using the same event name in the `<ProductSelector />` as well - for tracking clicks on the "Which one do I need" external link.

I'm happy to change it, though. Let's maybe ask @lucasartoni about the final confirmation.

Reply:
I don't see how that contradicts what I suggested. But yeah, let's see what Luca says.